robot_skills/src/robot_skills/robot.py

```diff
@@ -314,6 +314,21 @@ def handle_hardware_status(self, diagnostic_array):
             if name not in self._ignored_parts:
                 part.process_hardware_status(diagnostic_dict)
 
+    def move_to_inspect_pose(self, inspection_position):
+        """
+        This poses the robot for an inspect.
```

Comment:
Specify in you docstrings what these methods should return!

* Either: enforce that inheriting robots implement these methods by raising `NotImplementedErrors`
* Or: don't care whether they do; in that case: don't print this (either no log statements or only debug statements)